app/server.R

```diff
@@ -655,15 +682,17 @@ server <- function(input, output, session) {
           "Hospital Admissions" = "Hospitalizations"
         )
         DASH_SUFFIX <<- ""
-        updateTargetChoices(session, choices)
-      } else if (input$tabset == "evaluations_archive") {
+      } else if (input$tabset == paste0("evaluations", ARCHIVE_TAB_SUFFIX)) {
         choices <- list(
           "Incident Deaths" = "Deaths",
           "Incident Cases" = "Cases"
         )
-        DASH_SUFFIX <<- "_archive"
-        updateTargetChoices(session, choices)
+        DASH_SUFFIX <<- ARCHIVE_TAB_SUFFIX
+      } else {
+        return()
       }
+      updateTargetChoices(session, choices)
+      showScoreExplanation(session, input$scoreType, DASH_SUFFIX)
```

Comment:
(annotation for future-me)

this is the bit that (1) makes sure we only update targets and show scores on evaluations tabs, and (2) makes sure we show scores for the evaluations tabs